src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,13 @@
-export const TodoList = () => {};
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => <TodoInfo todo={todo} key={todo.id} />)}
```

Comment:
```suggestion
    {todos.map(todo => (
      <TodoInfo todo={todo} key={todo.id} />
    )}
```
 for me it is more readable